connectors/casa/main.py

```diff
@@ -88,13 +94,17 @@ def _fetch(offset, url):
         new_projects_offset = 0
         new_users_offset = 0
 
+    new_state = {
+        "fetch_more_users": new_fetch_more_users,
+        "fetch_more_projects": new_fetch_more_projects,
+        "users_offset": new_users_offset,
+        "projects_offset": new_projects_offset,
+    }
+
+    logging.info(f"Updated state: {new_state}, inserting {len(users)} users and {len(projects)} projects.")
+
```

Comment:
Perhaps it would be useful to log at the end if fetch more users or projects is true? Or is this something that gets automatically logged by Fivetran?

Reply:
Apologies, I meant `had_more`

Reply:
Good catch, added thanks!

Reply:
🙇